docs/testing-operators.md

```diff
@@ -28,6 +28,23 @@ The scorecard assigns points to each passing component of the scorecard. The tot
 
 **Note**: no explicit number of points or percentage is necessary to achieve before merging _yet_. These are suggestions to improve your operator.
 
+##### Running scorecard locally
```

Comment:
There should be a section about running the scorecard against an OpenShift cluster too. The two main differences are passing `"openshift"` instead of upstream and making sure `KUBECONFIG` is set in the environment.

Reply:
If there are no other differences, then I'll add this now. I haven't been able to test this with openshift yet